src/app/core/translations/pl_PL.ts

```diff
@@ -163,5 +163,9 @@ export default {
   CHOOSE_FROM_THE_LIST: 'Wybierz z listy',
   MESSAGE_SENT_ALERT_HEADER: 'Twoja odpowiedź została wysłana do osoby pomagającej',
   MESSAGE_SENT_ALERT_CONTENT: 'Potwierdzenie wysłania przesłaliśmy na Twój e-mail.',
+  ADVERTISEMENTS_LIST: 'Lista ogłoszeń',
```

Comment:
please add those keys at the end of a file

Reply:
Ten wątek został oznaczony jako Resolved, ale nie widzę, by uwaga Łukasza została zaadresowana.

Reply:
to bylo na końcu. Ale main branch ma nowe zmiany

Reply:
done again

Reply:
Cały czas ten klucz jest w środku.

Zmiany na `main` nie mają tu wiele do rzeczy, mogą co najwyżej wygenerować konflikt, ale nie wyrzucą Ci w diffie klucza na środek pliku.

Reply:
my bad, done